src/main/java/frc/team2412/robot/commands/diagnostic/LauncherDiagnosticCommand.java

```diff
@@ -1,10 +1,9 @@
 package frc.team2412.robot.commands.diagnostic;
 
+import edu.wpi.first.units.Angle;
```

Comment:
What's the import for? (And should the variable be changed to lowercase to prevent intellisense from doing something like this again?)